R/methods.R

```diff
@@ -1,5 +1,5 @@
 #' @export
-format.lint <- function(x, ...) {
+format.lint <- function(x, ..., width = getOption("lintr.format_width")) {
```

Comment:
I think we could write this as lintr_option("format_width") but that would not be user-friendly IINM.

Reply:
Yes, I agree. 

This convention also makes us consistent with how other tidyverse packages provide options (e.g. [pillar](https://pillar.r-lib.org/reference/pillar_options.html), [usethis](https://usethis.r-lib.org/reference/usethis_options.html), etc.).

Reply:
We should also respect `getOption("width")`, no?

Reply:
> We should also respect `getOption("width")`, no?

I think not really. `cat()` handles that -- I don't see a need to change the existing behavior in this regard. Normal terminal wrapping has worked fine so far for interactive usage (IMO). I think that would be a separate FR.

Reply:
@AshesITR I am going to merge this. Please create a follow-up if you feel strongly that `getOption("width")` should be respected here.